src/components/userCard.tsx

```diff
@@ -0,0 +1,27 @@
+import projectsCardIcon from "../assets/icons/projectsCardIcon.svg";
+import { UserCardProps } from "@/interfaces/userInterface";
+
+export function UserCard(props: UserCardProps) {
+  const { data } = props;
+  const imageUrl = data.imageUrl ?? "";
+
+  return (
+    <div className="bg-userCardBG w-48 h-52 rounded-2xl flex flex-col justify-center items-center shadow-md relative">
+      <img
+        src={imageUrl}
+        alt="userPhoto"
+        className="rounded-full h-16 w-16 border-white border-2"
```

Comment:
I see there are a bunch of places using the img tag with classes to make it round, just like an avatar. I suggest you create a shared style with that classes OR create a separate component called Avatar with styles pre-applied. Also, after creating it, replace the avatar in the sidebar with the component/style classes you implemented.

Reply:
i did a avatar component that the user has to pass the size (height and width) but the rest is the same